web/styles/inbox.css

```diff
@@ -787,3 +767,8 @@
         }
     }
 }
+
+.dropdown-widget-disabled-for-spectators #inbox-filter_widget {
+    cursor: not-allowed;
+    opacity: 0.5;
+}
```

Comment:
Let's use `.dropdown-widget-disabled-for-spectators .dropdown-widget-button` and move it near related CSS for `.dropdown-widget-button` in `app_components.css`.